docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* has a need to manage a meeting schedule
+* has a need to oversee the development of the product
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: Used in a corporate setting, our application integrates both a meeting schedule and a contact list for the user to 
+manage their product’s development more comprehensively and conveniently.
 
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+Priorities: 
+* `* * *` - High (must have)
+* `* *` - Medium (nice to have)
+* `*` - Low (unlikely to have) 
+
+| Priority | As a …​                               | I want to …​                     | So that I can…​                                                         |
+| -------- | ---------------------------------------- | --------------------------------------------------------------------------- | ---------------------------------- |
+| **Deliverables** |
```

Comment:
Maybe change the sections (_deliverables, contacts, meetings_) to their respective epics, following our meeting docs? E.g. this line of code can be "[EPIC] As a Product Manager, I can track my product’s development so that I can work better towards production deadlines. (Deliverables)"

Reply:
I did attempt to place the EPIC within each section, but github markdown does not allow you to create merged cells. So the sentence is wrapped within the a small single cell. I would suggest, if we want to include the epics, to create a separate table and reference the [EPICS] accordingly.

Reply:
I think your suggestion would work for now.